packages/Uploader/src/components/DropZone/DropZone.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import tokens from "@paprika/tokens";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import UploadIcon from "@paprika/icon/lib/Upload";
+import { UploaderContext } from "../../Uploader";
+import * as div from "./DropZone.styles";
+
+export default function DropZone() {
+  const I18n = useI18n();
+  const uc = React.useContext(UploaderContext);
+  const isDraggingOver = uc.isDraggingOver;
+  const uploadIconColor = isDraggingOver ? tokens.color.purpleDarken10 : tokens.color.blackLighten50;
+
+  /* eslint-disable jsx-a11y/anchor-is-valid */
+  const body = isDraggingOver ? (
+    I18n.t("uploader.drop_files")
+  ) : (
+    <React.Fragment>
+      {I18n.t("uploader.drop_files_here_or")}&nbsp;
+      <a
+        href="#"
+        onClick={() => {
+          uc.refInput.current.click();
+          return false;
+        }}
+      >
+        {I18n.t("uploader.choose_from_computer")}
+      </a>
+    </React.Fragment>
+  );
+  /* eslint-enable jsx-a11y/anchor-is-valid */
+
+  return (
+    <uc.FileInput>
+      <div.DropZone isDraggingOver={isDraggingOver}>
+        <UploadIcon size="32px" color={uploadIconColor} />
```

Comment:
Since you're using `tokens` already for the `color`, you could use one here for the size too... or else omit it and set the `font-size` for `[data-pka-ancor='icon']` to a token in the `DropZone` styles. Could also set the colour there, dependent on `isDraggingOver` if you want to really consolidate styling in one place.

Reply:
I've never liked "size" and "colour" props since they can just be targeted with css. However, since they are there, I'm gonna use 'em!

I can't use the `space` token for this, but can use `stylers/spacer`.